README.md

```diff
@@ -19,33 +19,33 @@ The module provides an access to INN-Reach.
+| KAFKA_HOST                    |           kafka           | Kafka broker hostname                                                                                                                                             |
+| KAFKA_PORT                    |           9092            | Kafka broker port                                                                                                                                                 |
+| KAFKA_SECURITY_PROTOCOL       |         PLAINTEXT         | Kafka security protocol used to communicate with brokers (SSL or PLAINTEXT)                                                                                       |
+| KAFKA_SSL_KEYSTORE_LOCATION   |             -             | The location of the Kafka key store file. This is optional for client and can be used for two-way authentication for client.                                      |
+| KAFKA_SSL_KEYSTORE_PASSWORD   |             -             | The store password for the Kafka key store file. This is optional for client and only needed if 'ssl.keystore.location' is configured.                            |
+| KAFKA_SSL_TRUSTSTORE_LOCATION |             -             | The location of the Kafka trust store file.                                                                                                                       |
+| KAFKA_SSL_TRUSTSTORE_PASSWORD |             -             | The password for the Kafka trust store file. If a password is not set, trust store file configured will still be used, but integrity checking is disabled.        |
+| LOG_HTTP                      |           false           | Enable logging of all requests and responses.                                                                                                                     |
+| DEFAULT_INTERVAL              |           95000           | Default interval in ms ( preferred not to increase more than this value )                                                                                         |
+ | MAX_FAILURE                   |            360            | Default max attempts                                                                                                                                              |
+| DEFAULT_OFFSET                |          latest           | Default kafka offset                                                                                                                                              |
+ | DEFAULT_CONCURRENCY           |      2                    | Default concurrency of kafka consumer                                                                                                                             |
+| INNREACH_TENANTS              |             -             | The list of Innreach tenants needs to mention with pipe symbol as delimiter. mod-innreach will only listen topics which are related to the tenants mentioned here |
```

Comment:
Can we have only that line change for adding the new INNREACH_TENANTS env variable ?

Reply:
Yes, changed it.